tests/test_core.py

```diff
@@ -52,41 +52,46 @@ def setUp(self):
         """
         super(AggregationUpdaterTestCase, self).setUp()
         self.agg_modified = now() - timedelta(days=1)
-        course_key = CourseKey.from_string('course-v1:edx+course+test')
+        self.course_key = CourseKey.from_string('course-v1:edx+course+test')
```

Comment:
Keep the diff minimal

```suggestion
        course_key = CourseKey.from_string('course-v1:edx+course+test')
        self.course_key = course_key
```